# Synthetic hybrids of six species
**Authors**: David Peris, William G. Alexander, Kaitlin J. Fisher, Ryan V. Moriarty, Mira G. Basuino, Emily J. Ubbelohde, Russell L. Wrobel, Chris Todd Hittinger

**Journal**: [Nature Communications](https://bit.ly/2We5YAB) 11:2085

**Year**: 2020

**Abstract**: Allopolyploidy generates diversity by increasing the number of copies and sources of chromosomes. Many of the best-known evolutionary radiations, crops, and industrial organisms are ancient or recent allopolyploids. Allopolyploidy promotes differentiation and facilitates adaptation to new environments, but the tools to test its limits are lacking. Here we develop an iterative method of Hybrid Production (iHyPr) to combine the genomes of multiple budding yeast species, generating Saccharomyces allopolyploids of at least six species. When making synthetic hybrids, chromosomal instability and cell size increase dramatically as additional copies of the genome are added. The six-species hybrids initially grow slowly, but they rapidly regain fitness and adapt, even as they retain traits from multiple species. These new synthetic yeast hybrids and the iHyPr method have potential applications for the study of polyploidy, genome stability, chromosome segregation, and bioenergy.

**[Nature Research Bioengineering Community blog entry](https://go.nature.com/2KF6tOK "Nature Research Bioengineering Community blog entry")**

## Successful crosses

- The frequency of hybrid production by each cross is recorded in the table [Raw_frequency\_crosses](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Raw_frequency_crosses.csv "Raw_frequency_crosses"). The data is used in *Figure 3A*

## Parent genome contribution

- The compressed file [winAvgDepth](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/winAvgDepth.rar "winAvgDepth") contains the individual files for the coverage values for each hybrid used to generate *Figures 4A, 4B, and 6B*
- The compressed file [nuclear_sppIDerDepthPlot](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/nuclear_sppIDerDepthPlot.rar "nuclear_sppIDerDepthPlot") contains the individual files plotting coverage across nuclear genomes of reference strains. Plots are generated by [sppIDer](http://bit.ly/2NacaXQ "sppIDer") and used in *Supplementary Figure 3*
- The compressed file [mito_sppIDerDepthPlot](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/mito_sppIDerDepthPlot.rar "mito_sppIDerDepthPlot") contains the individual files plotting coverage across mitochondrial genomes of references strains. Plots are generated by sppIDer and used in *Supplementary Figure 4*

## Chromosomal aberrations and chromosome composition

- The table [number_aberrations](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/number_aberrations.csv "number_aberrations") describes the chromosome losses, gains and aberrations inferred from sppIDer plots. The data is used in *Figure 2A and 6A*
- The compressed file [Chromosome_composition](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/Chromosome_composition.rar "Chromosome_composition") contains the tables with the chromosome composition by species and the total number of chromosomes for each hybrid. The data was inferred from the sppIDer plots. Data was used for generating *Figures 2B and 6C*

## Cell size and ploidy
- Raw brightfield microscopy images used to calculate 2-dimensional cell size are compressed in the [Cell_size\_raw\_images](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Cell_size_raw_images.tar.gz "Cell_size_raw_images") file.The estimated cell area for each strains is described in table [Cell_area\_estimations](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Cell_areas_raw_fiji_data.csv "Cell_area_estimations"). The data is used in *Figure 3B*.
- Cell volume estimations are described in table [Cell\_volume\_increase_data](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Cell_volume_increase_data.tsv "Cell_volume_increase_data"). The data is displayed in *Supplementary Table 2*
- Raw fluorescence values from flow cytometry are described in table [Flow_cytometry\_fluorescence\_data](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Flow_cytometry_fluorescence_data.csv "Flow_cytometry_fluorescence_data"). The data is used in Supplementary *Figures 5A, 5B, 5C*.

## Growth kinetic parameters
- Optical density measurements at 600nm are compressed in [Optical_density\_measurements](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Optical_density_measurements.rar "Optical_density_measurements"). Date_growthcurve.csv contains the measurements and Date_growthcurve_layout.csv has information about the strain and media contained in each well.
	- 20190819: *Figures 3C and 3D*.
	- 20180911: Supplementary *Figure 6*.
	- 20181113,20181130,20181210: *Figure 5A and Supplementary Figure 8*.

## Droptests
- Raw droptest pictures are compressed in [droptests](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/droptests.rar "Droptest pictures") file. The pictures were processed for *Figure 5B*

## Cassette coverage distribution
- Coverage values for chromosome IV extracted from sppIDer outputs and coverage values for xylose cassette sequence calculated from raw reads using [HybPiper](https://github.com/mossmatters/HybPiper "HybPiper repository") were described in the compressed files [ChrCovWindow](https://github.com/PerisD/SynHybrids/blob/master/hybpiper/ChrCovWindow.rar "ChrCovWindow") and [CassetteCov](https://github.com/PerisD/SynHybrids/blob/master/hybpiper/CassetteCov.rar "CassetteCov"), respectively. The data is used in *Supplementary Figure 7B*.

## Illumina reads

Whole genome sequences are deposited in SRA under accession number [Bioproject](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA476226 "Illumina reads") PRJNA476226

***

## Press Notes

- [CSIC - Abril 2020](https://bit.ly/35oI8WJ "CSIC Abril 2020") - Una nueva tecnología obtiene levaduras a la carta para mejorar la producción de bebidas alcohólicas y bioenergía

- [UW-Madison/GLBRC News - April 2020](https://bit.ly/3c0ImWU "UWMadison News April 2020") - Researchers create hybrids of six yeast species to combine useful traits

- [Anales Sectoriales - April 2020](https://bit.ly/3bVUJ6q "Anales Sectorial April 2020") - El CSIC desarrolla una nueva tecnología para obtener levaduras a la carta para mejorar la producción de bebidas alcohólicas

- [Renewable Energy Magazine - April 2020](https://bit.ly/3aUMYfI "Renewable Energy April 2020") - Levaduras a la carta para producir vino y biocarburantes

***

### Contact

[<img src="http://1000logos.net/wp-content/uploads/2017/03/LinkedIn-Logo.png" width="40"/>](https://bit.ly/2V9hFt4)

[<img src="https://www.uv.es/perisnav/Index/twitter-logo.png" width="50"/>](https://bit.ly/2WJ2Rm8)

[<img src="https://www.uv.es/perisnav/images/Mitogression.png" width="60"/>](https://www.uv.es/perisnav/)
