# Synthetic hybrids of six species
**Authors**: David Peris, William G. Alexander, Kaitlin J. Fisher, Ryan V. Moriarty, Mira G. Basuino, Emily J. Ubbelohde, Russell L. Wrobel, Chris Todd Hittinger

**Journal**: RRR

**Year**: 2020

AbstractXXX 

## Successful crosses

- The frequency of hybrid production by each cross is recorded in the table [Raw_frequency\_crosses](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Raw_frequency_crosses.csv "Raw_frequency_crosses"). The data is used in *Figure 3A*

## Parent genome contribution

- The compressed file [winAvgDepth](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/winAvgDepth.rar "winAvgDepth") contains the individual files for the coverage values for each hybrid used to generate *Figures 4A, 4B, and 6B*
- The compressed file [nuclear_sppIDerDepthPlot](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/nuclear_sppIDerDepthPlot.rar "nuclear_sppIDerDepthPlot") contains the individual files plotting coverage across nuclear genomes of reference strains. Plots are generated by [sppIDer](http://bit.ly/2NacaXQ "sppIDer") and used in *Supplementary Figure 3*
- The compressed file [mito_sppIDerDepthPlot](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/mito_sppIDerDepthPlot.rar "mito_sppIDerDepthPlot") contains the individual files plotting coverage across mitochondrial genomes of references strains. Plots are generated by sppIDer and used in *Supplementary Figure 4*

## Chromosomal aberrations and chromosome composition

- The table [number_aberrations](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/number_aberrations.csv "number_aberrations") describes the chromosome losses, gains and aberrations inferred from sppIDer plots. The data is used in *Figure 2A and 6A*
- The compressed file [Chromosome_composition](https://github.com/PerisD/SynHybrids/blob/master/sppIDer/Chromosome_composition.rar "Chromosome_composition") contains the tables with the chromosome composition by species and the total number of chromosomes for each hybrid. The data was inferred from the sppIDer plots. Data was used for generating *Figures 2B and 6C*

## Cell size and ploidy
- Raw brightfield microscopy images used to calculate 2-dimensional cell size are compressed in the [Cell_size\_raw\_images](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Cell_size_raw_images.tar.gz "Cell_size_raw_images") file.The estimated cell area for each strains is described in table [Cell_area\_estimations](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Cell_areas_raw_fiji_data.csv "Cell_area_estimations"). The data is used in *Figure 3B*.
- Cell volume estimations are described in table [Cell\_volume\_increase_data](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Cell_volume_increase_data.tsv "Cell_volume_increase_data"). The data is displayed in *Supplementary Table 2*
- Raw fluorescence values from flow cytometry are described in table [Flow_cytometry\_fluorescence\_data](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Flow_cytometry_fluorescence_data.csv "Flow_cytometry_fluorescence_data"). The data is used in Supplementary *Figures 5A, 5B, 5C*.

## Growth kinetic parameters
- Optical density measurements at 600nm are compressed in [Optical_density\_measurements](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/Optical_density_measurements.rar "Optical_density_measurements"). Date_growthcurve.csv contains the measurements and Date_growthcurve_layout.csv has information about the strain and media contained in each well.
	- 20190819: *Figures 3C and 3D*.
	- 20180911: Supplementary *Figure 6*.
	- 20181113,20181130,20181210: *Figure 5A and Supplementary Figure 8*.

## Droptests
- Raw droptest pictures are compressed in [droptests](https://github.com/PerisD/SynHybrids/blob/master/phenotyping/droptests.rar "Droptest pictures") file. The pictures were processed for *Figure 5B*

## Cassette coverage distribution
- Coverage values for chromosome IV extracted from sppIDer outputs and coverage values for xylose cassette sequence calculated from raw reads using [HybPiper](https://github.com/mossmatters/HybPiper "HybPiper repository") were described in the compressed files [ChrCovWindow](https://github.com/PerisD/SynHybrids/blob/master/hybpiper/ChrCovWindow.rar "ChrCovWindow") and [CassetteCov](https://github.com/PerisD/SynHybrids/blob/master/hybpiper/CassetteCov.rar "CassetteCov"), respectively. The data is used in *Supplementary Figure 7B*.

## Illumina reads

Whole genome sequences are deposited in SRA under accession number [Bioproject](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA476226 "Illumina reads") PRJNA476226

***

### Contact

[<img src="http://1000logos.net/wp-content/uploads/2017/03/LinkedIn-Logo.png" width="40"/>](https://goo.gl/xglg8H)

[<img src="http://www.stickpng.com/assets/images/580b57fcd9996e24bc43c53e.png" width="50"/>](https://goo.gl/OS0O2F)

[<img src="https://www.uv.es/perisnav/images/Mitogression.png" width="60"/>](https://www.uv.es/perisnav/)
